### ISSUE TITLE
Can't Input unpitched percussion from the Braille panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32323]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

There seems to be no way to input any notes on to the score from the Braille panel using sdfjkl 6-key entry for an unpitched percussion instrument.

STEPS:
1.  Create a score and add an instrument such as claves.
2. Enter a few notes on the score using A.
3. Tab to the Braille panel and try entering Braille notes for A4 quarter (k, djl), or A3 (jkl, djl), or A5 (jl, djl).
4. Try using the standard Braille input used by many countries of writing non pitched rhythms on Braille note C.  C4 quarter (k, fjkl) or C5 (jl, fjkl).
5. Try writing on the pitch that these display output on (think it is an E for claves).
6. Try also with each of the above leaving out the octave signs.

None of these work.  No one has found any way to input unpitched rhythm via the Braille panel.
Suggest they should be input on either note A or C without any octave signs, or A with a specific octave sign if this makes it more possible to get this working s it would be duplicating the score input, C with an octave sign would just seem wrong in Braille and the various different pitches they are output on is an issue anyway and would be hard to remember what they all are.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
See also 32324 confusing output in Braille of unpitched percussion.
